.github/workflows/universalnrp-test.yml

```diff
@@ -52,25 +52,27 @@ jobs:
             [
               { "os": "almalinux", "version": "9" },
               { "os": "centos", "version": "8" },
-              { "os": "debian", "version": "10" },
               { "os": "debian", "version": "11" },
               { "os": "debian", "version": "12" },
               { "os": "mariner", "version": "2" },
               { "os": "oraclelinux", "version": "8" },
-              { "os": "rhel", "version": "7" },
               { "os": "rhel", "version": "8" },
               { "os": "rhel", "version": "9" },
               { "os": "rockylinux", "version": "9" },
               { "os": "sles", "version": "15" },
               { "os": "ubuntu", "version": "20.04" },
               { "os": "ubuntu", "version": "22.04" }
             ]
-          # { "os": "amazonlinux", "version": "2" },
-          # { "os": "centos", "version": "7" },
-          # { "os": "oraclelinux", "version": "7" },
-          # { "os": "sles", "version": "12" },
-          # { "os": "ubuntu", "version": "16.04" },
-          # { "os": "ubuntu", "version": "18.04" },
+          # { "os": "amazonlinux", "version": "2" }, [TODO: Onboard]
+          # { "os": "azurelinux", "version": "3" }, [TODO: Onboard]
+          # { "os": "centos", "version": "7" }, [EOL June 30, 2024]
+          # { "os": "debian", "version": "10" }, [EOL June 30, 2024]
+          # { "os": "mariner", "version": "1" }, [EOL July 31, 2023]
+          # { "os": "oraclelinux", "version": "7" }, [EOL December 31, 2024]
+          # { "os": "rhel", "version": "7" }, [EOL June 30, 2024]
+          # { "os": "sles", "version": "12" }, [EOL October 31, 2024]
+          # { "os": "ubuntu", "version": "16.04" }, [TODO: Onboard EOL April 21, 2021]
+          # { "os": "ubuntu", "version": "18.04" }, [EOL May 31, 2023]
```

Comment:
Can we simply remove these commented out lines, as in module-test.yml? No need to list them here really